### ISSUE TITLE
Cannot find VEP script under

### ISSUE THREAD

**Issue by jingydz** [mskcc/vcf2maf#314]
$ perl vcf2maf.pl --input-vcf xxx.snp.vcf --output-maf test.maf --ref-fasta ../pig.dna.toplevel.fa
STATUS: Running VEP and writing to: xxx.snp.vep.vcf
ERROR: Cannot find VEP script under: /yyy/miniconda3/bin

**Comment by allaway** [comment]
Hello @jingydz, you can fix this using the `--vep-path` parameter. 

For example, if you installed vep into a conda environment you could run: 

`find /. -name '*vep*'` to locate vep. 
You then might see something like 
`/opt/conda/envs/vcf2maf/bin/vep`, which would mean you should run vcf2math with: `--vep-path /opt/conda/envs/vcf2maf/bin/`

**Comment by ckandoth** [comment]
Yes, thanks for your answer. Path to VEP needs to be specified if installed in a different path than recommended by docs.
